src/main/java/com/ncov/module/service/MaterialRequiredService.java

```diff
@@ -0,0 +1,107 @@
+package com.ncov.module.service;
+
+import com.baomidou.mybatisplus.core.conditions.query.LambdaQueryWrapper;
+import com.baomidou.mybatisplus.core.metadata.IPage;
+import com.baomidou.mybatisplus.extension.plugins.pagination.Page;
+import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
+import com.google.common.base.Joiner;
+import com.ncov.module.controller.dto.MaterialDto;
+import com.ncov.module.controller.request.material.MaterialRequest;
+import com.ncov.module.controller.resp.material.MaterialResponse;
+import com.ncov.module.entity.MaterialRequiredEntity;
+import com.ncov.module.mapper.MaterialRequiredMapper;
+import com.ncov.module.security.UserContext;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * 物料寻求服务
+ * @author lucas
+ */
+@Slf4j
+@Service
+public class MaterialRequiredService extends ServiceImpl<MaterialRequiredMapper, MaterialRequiredEntity> {
+
+    @Autowired
```

Comment:
In spring boot 2, `@Autowired` is redundant when class is annotated as a component/service/controller. Combine with lombok `@AllArgsConstructor` will make this easier. Just suggesting, not critical.

Reply:
thanks